### PR TITLE
[#39] Properly defaulting user roles

### PR DESCRIPTION
**Pull request by brianghig**
if they are not provided when initializing a new user. Added some unit tests to cover the initializeNewUser method.

### CODE REVIEW

File: src/server/app/admin/services/users.authentication.server.service.spec.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+		organization: key + ' Organization'
+	};
+}
+
+function validateDefaultRoles(updatedUser) {
+	var keys = _.keys(testDefaultRoles);
+
+	should.exist(updatedUser.roles);
+
+	_.forEach(keys, function(key) {
```
Comment (reblace) [inline src/server/app/admin/services/users.authentication.server.service.spec.js]: We started using arrow functions in the server code everywhere. You should refactor these to use them.

**Review by reblace — CHANGES_REQUESTED** [review]
It makes me so happy that you added a spec for this. Can you make the arrow function refactor quick? and then I'm fine w/merging this in.

File: src/server/app/admin/services/users.authentication.server.service.spec.js (hunk 1 above)
Comment (brianghig) [reply src/server/app/admin/services/users.authentication.server.service.spec.js]: Done! Thanks for the suggestion.

**Comment by brianghig** [comment]
@reblace @bodnarbm so awesome having Travis-CI build that automatically to catch a silly error. Thanks for setting that up!

**Pull request merged**